src/app/clients/page.tsx

```diff
@@ -9,7 +8,7 @@ const dataFetch: () => Promise<Schema["clients"][]> = async () => {
 };
 
 const Clients: () => Promise<React.ReactElement> = async () => {
-    const data: Datum[] = await dataFetch();
+    const data = await dataFetch();
```

Comment:
Why was the type removed?

Reply:
type inference gives it a more specific type and results in no type errors, so having no explicit type would be preferred.